packages/shared/locales/en.json

```diff
@@ -1049,14 +1049,20 @@
         "creatingProfile": "Creating profile, please wait...",
         "fundMigration": "Fund migration",
         "accountRemoved": "This wallet is hidden. Unhide it to perform transfers.",
-        "receivingTo": "Receiving to {account}",
-        "sendingFrom": "Sending from {account}",
-        "receivedTo": "Received to {account}",
-        "sentFrom": "Sent from {account}",
+        "receivingFrom": "Receiving from {account}",
+        "sendingTo": "Sending to {account}",
+        "receivedFrom": "Received from {account}",
+        "sentTo": "Sent to {account}",
+        "transferringFrom": "Transferring from {account}",
+        "transferringTo": "Transferring to {account}",
+        "transferFrom": "Transfer from {account}",
```

Comment:
Should we match the progressive-/past-tense pair for transfers as it is for sending and receiving? E.g. `Transferring from A` -> `Transferred from A` rather than `Transferring from A` -> `Transfer from A`. It's more consistent with `Receiving from A` -> `Received from A`.

Reply:
Good point, although the longer text means the probability of more text strings wrapping onto new line, which is a pain.

Reply:
Yes, that's definitely true - I suppose the `Transferring from ...` would only be painful while the message is getting confirmed. This inconsistency isn't much of a problem though imo, it still works as `Transfer from ...`.